mutablesecurity/solutions/implementations/clamav/meta.yaml

```diff
@@ -0,0 +1,10 @@
+full_name: ClamAV
+description: ClamAV is an open source (GPLv2) anti-virus toolkit, designed especially for e-mail scanning on mail gateways. It provides a number of utilities including a flexible and scalable multi-threaded daemon, a command line scanner and advanced tool for automatic database updates. The core of the package is an anti-virus engine available in a form of shared library.
```

Comment:
I think that the effective open source license is not that relevant. In addition, the description does not inform the user about the main usage of what we implement in this module: an antimalware software capable of scanning the file-system.

Reply:
Done.